src/native-stack/types.tsx

```diff
@@ -112,6 +112,15 @@ export type NativeStackNavigationOptions = {
    * @platform ios
    */
   disableBackButtonMenu?: boolean;
+  /**
+   * How the back button behaves by default (when not customized). Available on iOS>=14, and is used only when none of: `backTitleFontFamily`, `backTitleFontSize`, `disableBackButtonMenu` or `backTitle` is set.
+   * The following values are currently supported (they correspond to https://developer.apple.com/documentation/uikit/uinavigationitembackbuttondisplaymode?language=objc):
+   * - "default" – show given back button previous controller title, system generic or just icon based on available space
+   * - "generic" – show given system generic or just icon based on available space
+   * - "minimal" – show just an icon
+   * @platform ios
```

Comment:
I don't know if suggesting `iOS` is good idea right now, since we're doing `@platform android` / `@platform ios` everywhere. Let's maybe do that in separate PR.

Reply:
I agree with @tboba almost always we use `@platform ios` there is just one case when we write `iOS`

Reply:
That's a subject to change then, but it might be done in separate PR, I do not mind.